src/Gui/DAGView/DAGModel.cpp

```diff
@@ -490,39 +482,11 @@ void Model::updateSlot()
     }
   }
 
-  //apply filters.
+  //sync scene items to graph vertex dagVisible. (bogus)
```

Comment:
What does the "bogus" comment mean here?

Reply:
Removed.